identity/sharedid.md

```diff
@@ -132,9 +145,215 @@ by the publisher.
+
+If there are no custom configurations, then just include the script and it'll use the default values.
+
+```
+<script type="text/javascript" src="//myserver.com/pubcid.min.js"></script>
+```
+
+If custom configurations are needed, define the pubcid_options object before inclusion of the script. Below is an example to switch from using local storage to cookie:
+
+```
+<script type="text/javascript">
+   window.pubcid_options = {type: 'cookie'};
+</script>
+<script type="text/javascript" src="//myserver.com/pubcid.min.js"></script>
+```
+
+#### Configuration
+
+Configuration
+Below are the available configuration options for the PubCID script.
+
+{: .table .table-bordered .table-striped }
+| Parameter Name | Type | Description | Example |
+| --- | --- | --- | --- | 
+| create | boolean | If true, then an id is created automatically by the script if it's missing. Default is true. If your server has a component that generates the id instead, then this should be set to false
+ | `TRUE` |
+| expInterval | decimal | Expiration interval in minutes. Default is 525600, or 1 year
+ | `525600` |
+| extend | boolean | If true, the the expiration time is automatically extended whenever the script is executed even if the id exists already. Default is true. If false, then the id expires from the time it was initially created. | For publisher server support only.  If true, the publisher's server will create the (pubcid) cookie.  Default is true. | `TRUE` |
+| params.pixelUrl | Optional | String | For publisher server support only. Where to call out to for a server cookie. | `/wp-json/pubcid/v1/extend/`
+| type | string | Type of storage. It's possible to specify one of the following: 'html5', 'cookie'. Default is 'html5' priority, aka local storage, and fall back to cookie if local storage is unavailable. | If true, the expiration time of the stored IDs will be refreshed during each page load.  Default is false. | `cookie` |
+
+#### Example Configurations
+
+Always use cookies and create an ID that expires in 30 days after creation.
+
+```
+{ 
+    type: 'cookie',
+    extend: false,
+    expInterval: 43200
+}
+```
+
+Using a SharedId Server implementation, create the cookie once, which will be allowed to expire before it is created again.
+
+```
+{ 
+    type: 'cookie',
+    pixelUrl: '/wp-json/pubcid/v1/extend/',
+    create: false,
+    extend: false
+}
+
+```
+
+### SharedId Server
+
+Add server-side support for SharedId to better handle the ever-increasing restrictions on cookies in modern web browsers by having the SharedId cookie written and extended by your web server.
+
+#### CMS
+
+Plugins are available for Wordpress and Drupal. Because CMS can cache pages to improve scalability, it's impractical to extend cookies during page generation. Instead the plugins add a dynamic endpoint that serves back a blank pixel, and updates cookies at the same time. The client side script therefore needs one additional
```

Comment:
one additional ... what?

Reply:
"parameter for this URL. Please consult the corresponding plugin documents for default values."

Reply:
added